script/Ciel/Day 6/06_01_CIEL06_7.txt

```diff
@@ -351,12 +411,13 @@ Truly, as though she was now pondering whether to lick her lips.
 [sha:93b4ab6a6f1467d273a93fe0325a40d80e076623]{
 -- Page 279, Offset 13894.
 -- マーリオゥの驚愕は如何ほどのものか。
-How great was Mario's astonishment?
+Just how surprised must Mario be?
 }
 [sha:dcf2834550cf805c37ff5f08b1c31d245b7fb535]{
 -- Page 279, Offset 13895.
 -- まさか天地が逆さになるほどの奇蹟を、こんな極東の地で耳にしようとは。
-Like catching wind of a miracle, that in this Far East land, Heaven and Earth have been turned upside down.
+What he's just heard, all the way out here in this Far East land,
```

Comment:
"Fear East land" is a little unusual. I'd expect it to be either "Far Eastern land" or just "in the Far East"